src/extension/extension.ts

```diff
@@ -317,6 +317,43 @@ export function activate(context: vscode.ExtensionContext) {
         }),
     );
 
+    // create a pet coding session on vscode start
+    const autoStart = vscode.workspace
+        .getConfiguration('vscode-pets')
+        .get<Boolean>('autoStartPetCodingSession');
+    if (autoStart) {
+        if (
```

Comment:
Is this code different to `vscode-pets.start` ? Can it be refactored into a function to avoid duplication

Reply:
I tried to call the function but it carried an error - will retry

Reply:
Ok fixed now